.github/workflows/gke.yaml

```diff
@@ -0,0 +1,120 @@
+name: ConformanceGKE
+
+on:
+  pull_request_target:
+    types:
+      - "labeled"
+  #pull_request: #uncomment for testing
+  #  types:
+  #    - "labeled"
+  push:
+    branches:
+      - master
+
+env:
+  clusterName: cilium-cli-ci-${{ github.run_number }}
+  zone: us-west2-a
+
+jobs:
+  installation-and-connectivitiy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    if: ${{ github.event.label.name == 'ci-run/gke' || github.event_name == 'push' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set image tag
+        id: vars
+        run: |
+          if [ ${{ github.event.pull_request.head.sha }} != "" ]; then
+            echo ::set-output name=tag::${{ github.event.pull_request.head.sha }}
+          else
+            echo ::set-output name=tag::${{ github.sha }}
+          fi
+
+      - name: Install Cilium CLI
+        run: |
+          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.4/cilium-linux-amd64.tar.gz
+          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/bin
+          rm cilium-linux-amd64.tar.gz
+
+      - name: Set up gcloud
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_PR_SA_KEY }}
+          export_default_credentials: true
+
+      - name: gcloud info
+        run: |
+          gcloud info
+
+      - name: Create GKE cluster
+        run: |
+          gcloud container clusters create ${{ env.clusterName }} \
+            --preemptible \
```

Comment:
I'll be curious whether `--preemptible` has any impact on CI stability.  Hopefully no impact.

Reply:
Not sure TBH. We will only know if we run it for a bit and get data.

Reply:
It will have some impact and we can always just re-run, but yea, from my testing the clusters are sufficiently short-lived that it basically never is an issue.  I suppose we'll have to see how it performs for real.